sql-select/five-customer-emails.sql

```diff
@@ -0,0 +1,3 @@
+select "customerId","firstName","lastName",'email'
+from "customers"
+limit 5;
```

Comment:
Please take a screenshot of every query and verify that the outputs look correct.

---

![image](https://user-images.githubusercontent.com/16143771/144687108-42bc9c09-338d-413e-ad01-37535eadcdce.png)

Reply:
@GustavoAyala1 Do these rows look like email addresses?

---

![](https://user-images.githubusercontent.com/80282083/144687919-a8096843-d6eb-4753-b950-c72d9db12fd3.png)

---

![image](https://user-images.githubusercontent.com/16143771/144691466-7fded34a-b8ac-4e32-9425-141a5667d62c.png)